jakarta.batch.tck.api/src/main/java/ee/jakarta/tck/batch/api/vehicle/ejb/EJBVehicleRunner.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Eclipse Foundation.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ee.jakarta.tck.batch.api.vehicle.ejb;
+
+import ee.jakarta.tck.batch.api.PropertyKeys;
+import java.util.Properties;
+
+import ee.jakarta.tck.batch.api.Reporter;
+import ee.jakarta.tck.batch.api.extensions.util.TSNamingContext;
+import ee.jakarta.tck.batch.api.vehicle.VehicleRunnable;
+
+public class EJBVehicleRunner implements VehicleRunnable {
+
+    @Override
+    public void run(Runnable test, Properties p) throws Exception {
+        EJBVehicleRemote ref = null;
+        String sEJBVehicleJndiName = p.getProperty(PropertyKeys.EJB_VEHICLE_JNDI_NAME, "java:global/jbatch-test-package-all/EJBVehicle");
+        TSNamingContext jc = new TSNamingContext();
```

Comment:
I just took inspiration from the implementation of the EJB vehicle in Jakarta TCK, so I copied TSNamingContext from there. Now when I look at it we really don't need the extra functionality so I can refactor this to use the plain InitialContext.

Reply:
In the new PR #40, I decided to keep this class because it wraps the code required to cast the object retrieved from lookup. We could use plain InitialContext but then the code to cast to `EJBVehicleRemote` would have to be added here.